src/commands/force/user/password/generate.ts

```diff
@@ -34,6 +43,12 @@ export class UserPasswordGenerateCommand extends SfdxCommand {
 
   public async run(): Promise<PasswordData[] | PasswordData> {
     this.usernames = (this.flags.onbehalfof as string[]) ?? [this.org.getUsername()];
+    if (this.flags.length > 20 || this.flags.length < 8)
```

Comment:
this is all unnecessary if you use the flag properties.  Then you can remove those messages, too.

Also, why not > 20?

Reply:
changing max to 1000.